docs/source/configuration/tracing.mdx

```diff
@@ -105,8 +105,8 @@ telemetry:
   tracing:
     jaeger:
       agent:
-        # Either 'default' or a socket address 
-        endpoint: default
+        # Either 'default' or an url 
+        endpoint: docker_jaeger:14268
```

Comment:
It's not an url if there is no scheme, but the world has moved on...

Reply:
well it accepts a proper URL too: https://github.com/apollographql/router/pull/1018/files#diff-57525742878233b1b5baf1a3e2d1d87d049bffb7b0da7474426356d96f94ad2fR40
It's just that people will tend to write host:port anyway, and the URL parser would interpret `docker_jaeger` as scheme and `14268` as path :sweat_smile: